src/components/Banner/banner.module.css

```diff
@@ -0,0 +1,45 @@
+.wrapper {
+  position: relative;
+  width: 100%;
+  background-color: #f8f5f1;
+  background-image: url("../../images/gradient.png");
+  height: var(--banner-height, 300px);
+  padding-left: 30px;
+  padding-right: 30px;
+  display: flex;
+  justify-content: flex-start;
+  align-items: flex-start;
+  flex-direction: column;
+}
+
+.container {
+  position: relative;
+  width: 100%;
+  overflow: hidden;
+}
+
+.logo {
+  height: 66%;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+}
+
+.vector {
+  width: 100%;
+  position: absolute;
+  left: 0;
+}
+
+.welcomeMessage {
+  position: absolute;
+  right: 8%;
+  transform: translateY(-50%);
+  padding-top: 27%;
```

Comment:
positioning with translate is a sign that you messed up something elese

Reply:
done